src/geo/LatLng.js

```diff
@@ -17,7 +17,7 @@ import {toLatLngBounds} from './LatLngBounds';
  *
  * ```
  * map.panTo([50, 30]);
- * map.panTo({lat: 50, lng: 30});
+ * map.panTo({lat: 50, lng: 30}); // Note: `lon` can be used interchangably with `lng` in objects.

```

Comment:
I think it's clearer with an example:

```suggestion
 * map.panTo({lat: 50, lng: 30});
 * map.panTo({lat: 50, lon: 30});
```